Assets/Scripts/Interpreter/Lib/Tower.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Interpreter.Eval.Type;
+using UnityEngine;
+
+namespace Interpreter.Lib.Tower
+{
+    public class CreateTower : MonoBehaviour
+    {
+        public GameObject obj;
+
+        public Nil Exec(Number x, Number y)
+        {
+            //@TODO: check for resources (waiting @ClaraOMello)
+
+            float xPosition = (float) x.AsNumber();
+            float yPosition = (float)y.AsNumber();
+
+            GameObject tower = Instantiate(obj);
+            tower.transform.position = new Vector3(xPosition, yPosition);
+
+            return new Nil();

```

Comment:
Okay, we may indeed need another class to abstract this away. Let's just wait a few more use cases to try to better identify the pattern.

Reply:
Also, if you want to add a method `AsFloatNumber` to the `Value` class, be my guest.

Reply:
tysm